core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -155,8 +157,10 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.startObject();
             if (item.isFailure()) {
                 ElasticsearchException.renderThrowable(builder, params, item.getFailure());
+                builder.field("status", ExceptionsHelper.status(item.getFailure()).getStatus());
```

Comment:
`status` must be declared in the static inner `Fields` class like `Fields.RESPONSES`